components/lvgl_esp32_drivers/lvgl_tft/ili9488.c

```diff
@@ -122,6 +122,9 @@ void ili9488_flush(lv_disp_drv_t * drv, const lv_area_t * area, lv_color_t * col
 
     lv_color16_t *buffer_16bit = (lv_color16_t *) color_map;
     uint8_t *mybuf = (uint8_t *) heap_caps_malloc(3 * size * sizeof(uint8_t), MALLOC_CAP_DMA);
+    while (mybuf == NULL) {
+        mybuf = (uint8_t *) heap_caps_malloc(3 * size * sizeof(uint8_t), MALLOC_CAP_DMA);
```

Comment:
How about a `do { } while(mybuf == NULL)` loop to avoid repeating the same line twice?

Reply:
I agree :+1: I will fix it. Thought about adding some kind of warning message as well. Without it the UI would just freeze without showing any kind of hint why.
So maybe change it to something like: 
`do { ... if (mybuf == NULL) {ESP_LOGW()} } while(mybuf == NULL)`. Still not beautiful either, but at least some debugging info.